### DISCUSSION TITLE
Does Kiali accept traces sent to an external Jaeger in Open Telemetry format?

### DISCUSSION THREAD

**Discussion by thomaspaulin** [kiali/kiali#6409] [Q&A]
All the examples of Jaeger with Istio that I've found use the all-in-one Jaeger, however, I have one which separates out into its constituent components using an Open Telemetry agent injected into my application container, an Open Telemetry collector DaemonSet, a Jaeger collector DaemonSet, and the Jaeger query service. With this setup my traces don't show in Kiali's Traces tab, but if I switch to the all-in-one Jaeger they do, from this I gather it must be that I'm missing certain attributes, or the otel format is incompatible wit the Zipkin format that Kiali seems to want. Can anyone talk to Open Telemetry support in Kiali?

Jaeger: 1.46.0
Kiali: 1.50.0 (97bc6bf1a41d9fb231ccca0781eb575c91b451d1)
Istio: 1.13.3 
Prometheus: 2.28.1 
Kubernetes: 1.21.12

**Comment by jshaughn** [comment]
Hmm, Kiali is just a consumer of traces and shouldn't care how the trace store is populated.  As long as the Jaeger querying API is unchanged I would have expected it to work.  Anything in the Kiali logs?  Is Kiali showing Jaeger as unavailable on the masthead?
